docs/index.md

```diff
@@ -278,7 +277,6 @@ output "skysql_credentials" {
   sensitive = true
 }
 
-
 # Example how you can generate a command line for the database connection
 output "skysql_cmd" {
   value = "mariadb --host ${data.skysql_service.default.fqdn} --port 3306 --user ${data.skysql_service.default.service_id} -p --ssl-ca ~/Downloads/skysql_chain_2022.pem"
```

Comment:
This command line is not correct for new databases launched.

Reply:
Yes, I noticed that using the pem chain did not get me access.  What should be the correct command here?